diff -urN autorpm-3.3.2.orig/autorpm.cron autorpm-3.3.2/autorpm.cron
--- autorpm-3.3.2.orig/autorpm.cron	2003-05-19 16:38:59.000000000 +0200
+++ autorpm-3.3.2/autorpm.cron	2004-07-04 16:08:48.000000000 +0200
@@ -15,5 +15,5 @@
 # looking for updates (backgrounds the process to avoid delaying
 # other cron jobs)
 export LC_ALL=C
-/usr/sbin/autorpm --notty "auto --delay=7200" &
+#/usr/sbin/autorpm --notty "auto --delay=7200" &
 
diff -urN autorpm-3.3.2.orig/autorpm.d/sample_configs/autorpm.conf-sample autorpm-3.3.2/autorpm.d/sample_configs/autorpm.conf-sample
--- autorpm-3.3.2.orig/autorpm.d/sample_configs/autorpm.conf-sample	2004-04-05 01:02:30.000000000 +0200
+++ autorpm-3.3.2/autorpm.d/sample_configs/autorpm.conf-sample	2004-07-04 16:08:19.000000000 +0200
@@ -27,12 +27,13 @@
 # This automatically determines the version of Red Hat Linux
 # You have... you can comment this out and define it yourself
 # if you want to
-Eval_Var("RHVersion", "sed 's/\(Red Hat Linux \)\?release \([^ ]*\) (.*)/\2/' /etc/redhat-release");
+#Eval_Var("RHVersion", "sed 's/\(Red Hat Linux \)\?release \([^ ]*\) (.*)/\2/' /etc/redhat-release");
 #Set_Var("RHVersion", "6.1");
 
 # Look for official Red Hat updates
 # (won't automatically install anything unless you edit the file)
-Config_File("/etc/autorpm.d/redhat-updates.conf");
+#Config_File("/etc/autorpm.d/redhat-updates.conf");
+Config_File("/etc/autorpm.d/pld-updates.conf");
 ########################## END Red Hat Linux #################################
 
 ######################## BEGIN Fedora Linux #################################
